package.json

```diff
@@ -5,6 +5,7 @@
 	"scripts": {
 		"predev": "bun prepare-font.js",
 		"dev": "SENTRY_SUPPRESS_TURBOPACK_WARNING=1 next dev --turbo",
+		"debug": "LOGLEVEL=debug bun run dev",
```

Comment:
```suggestion
		"dev-debug": "LOGLEVEL=debug bun run dev",
```

Reply:
🍠 imo

I thought it might be better to use `dev-debug` explicitly, as just `debug` makes it hard to distinguish what kind of `debug` it is.

example: https://github.com/vercel/next.js/blob/d6dd69f210eb36360e6b4544a5f7a6918e8bdb63/package.json#L18

Reply:
thanks!